src/pkg/services/m365/api/lists_test.go

```diff
@@ -55,3 +57,66 @@ func (suite *ListsAPIIntgSuite) TestLists_PostDrive() {
 	_, err = acl.PostDrive(ctx, siteID, driveName)
 	require.ErrorIs(t, err, graph.ErrItemAlreadyExistsConflict, clues.ToCore(err))
 }
+
+func (suite *ListsAPIIntgSuite) TestLists_GetListById() {
+	t := suite.T()
+
+	ctx, flush := tester.NewContext(t)
+	defer flush()
+
+	var (
+		acl            = suite.its.ac.Lists()
+		siteID         = suite.its.site.id
+		listID         = "integration-test-list"
+		textColumnName = "ItemName"
+		itemName       = "new item"
+	)
+
+	// <------ Setup ---->
+	// create list
+	list := models.NewList()
+	list.SetDisplayName(&listID)
+
+	_, err := acl.Stable.Client().Sites().BySiteId(siteID).Lists().Post(ctx, list, nil)
```

Comment:
To keep with code standards, please multi-line long func chains like this.
```suggestion
	_, err := acl.Stable.
	  Client().
	  Sites().
	  BySiteId(siteID).
	  Lists().
	  Post(ctx, list, nil)
```